web/views/pages/index.ejs

```diff
@@ -266,12 +266,12 @@
                                 <!--<span class="premium"><i class="fa fa-lock" aria-hidden="true"></i> premium</span>-->
                                 <br />
                                 <small>
-                                    <i class="fa fa-download" aria-hidden="true"></i> <a href="https://pro.electricitymap.org/carts/new" target="_blank"><%= __('country-history.downloaddataset') %></a>
+                                    <i class="fa fa-download" aria-hidden="true"></i> <a href="https://pro.electricitymap.org/?utm_source=https%3A%2F%2Fwww.electricitymap.org&utm_medium=referral&utm_campaign=country_panel" target="_blank"><%= __('country-history.downloaddataset') %></a>
```

Comment:
Now we redirect to the homepage instead of /carts/new. Is that intended? Both are fine for me.